### ISSUE TITLE
Nicol Bolas, God-Pharaoh not working fine

### ISSUE THREAD

**Issue by DoidArthas** [WagicProject/wagic#1063]
The first ability of Nicol Bolas, God-Pharaoh says "without paying its mana cost", but in wagic, to play that card I must pay the cost.

**Comment by Vitty85** [comment]
Yes, you are right, i corrected that card in this way:

`[card]`
`name=Nicol Bolas, God-Pharaoh`
`auto=counter(0/0,7,loyalty)`
`aicode=activate transforms((,newability[all(*[zpos=oppofindfirsttypenonland]|opponentlibrary) moveto(myexile) and!( transforms((,newability[zerocast],newability[canplayfromexile],newability[phaseaction[my endofturn once checkex] moveto(ownerExile)],newability[all(*[zpos<=oppofindfirsttypenonland]|opponentlibrary) moveto(ownerExile)])) ueot )!])) oneshot`
`auto={C(0/0,2,Loyalty)}:name(+2: Exile cards from top and may cast it) reveal:1 revealzone(opponentlibrary) revealuntil(*[-land]|opponentlibrary) optionone name(Move to your exile for cast) target(*[-land]|reveal) moveto(myexile) and!( transforms((,newability[zerocast],newability[canplayfromexile],newability[phaseaction[my endofturn once checkex] moveto(ownerExile)])) ueot )! optiononeend optiontwo name(Exile other cards) all(*|reveal) moveto(ownerExile) oneshot optiontwoend revealend`
`auto={C(0/0,1,Loyalty)}:name(+1: Opponent exiles two cards from hand) ability$!name(Move to exile) name(Move to exile) moveto(exile) notatarget(<2>*|myhand)!$ opponent`
`auto={C(0/0,-4,Loyalty)}:name(-4: Deals 7 damage to opponent) damage:7 opponent`
`auto={C(0/0,-4,Loyalty)}:name(-4: Deals 7 damage to opponent creature) damage:7 target(creature|opponentbattlefield)`
`auto={C(0/0,-12,Loyalty)}:name(-12: Exile all non-land permanents) moveTo(exile) all(*[-land]|opponentbattlefield)`
`text=+2: Target opponent exiles cards from the top of his or her library until he or she exiles a nonland card. Until end of turn, you may cast that card without paying its mana cost. -- +1: Each opponent exiles two cards from his or her hand. -- -4: Nicol Bolas, God-Pharaoh deals 7 damage to target opponent or creature an opponent controls. -- -12: Exile each nonland permanent your opponents control.`
`mana={4}{U}{B}{R}`
`type=Legendary Planeswalker`
`subtype=Bolas`
`[/card]`

Basically you will exile the non land card into your exile and until the end of turn you will be able to cast for zero from there. At the end of your turn that card will go back to owner exile and you won't be able to cast it anymore.

Hope it's ok for you, , i will add this fix in the official release ASAP. 

See you.

P. S.
If you can do some test too i will appreciate.

**Comment by Vitty85** [comment]
Closed in last official Wagic release update: https://github.com/WagicProject/wagic/releases/tag/wagic-v0.23.1
